infra/concourse/pipelines/terraform-google-vm.yml

```diff
@@ -17,15 +17,15 @@ resources:
     type: docker-image
     source:
       repository: gcr.io/cloud-foundation-cicd/cft/lint
-      tag: 1.0.0
+      tag: 2.5.0
       username: _json_key
       password: ((sa.google))
 
   - name: integration-test-image
     type: docker-image
     source:
       repository: gcr.io/cloud-foundation-cicd/cft/kitchen-terraform
-      tag: 1.2.0
+      tag: 2.4.0
```

Comment:
This should be 2.3.0; 2.4.0 is not ready for use yet.

Reply:
Done